modules/ai/assets/js/editor/pages/form-layout/components/prompt-form.js

```diff
@@ -145,12 +146,12 @@ const PromptForm = forwardRef( ( {
 
 			<EnhanceButton
 				size="small"
-				disabled={ isInteractionsDisabled }
+				disabled={ isGenerateDisabled || '' === prompt }
```

Comment:
```suggestion
				disabled={ isGenerateDisabled }
```

redundant?

Reply:
In case you have an attachment but not a prompt, you can generate but not enhance the prompt.